doc/plugins.dox

```diff
@@ -170,14 +171,22 @@ Geany's website at http://www.geany.org/Support/BuildingOnWin32.
 
 @section helloworld "Hello World"
 
-When writing a plugin, you will find a couple of functions or macros which are mandatory
-and some which are free to use for implementing some useful feature once your plugin
-becomes more powerful like including a configuration or help dialog.
+@note This section describes the new entry points for plugins introduced with Geany 1.26. A short
+summary of the legacy entry points is given at page @ but it should not be used any more.
```

Comment:
Might be nothing if it didn't strike @elextr, but for me "but *it* should not" if wrong, it should be plural as we're talking about "entry point*s*", shouldn't it?
"… but they should not be used for new plugins any more."